### PR TITLE
Refactor AlethZero to be modular; isolate the AllAccounts module.

### PR DESCRIPTION
**Pull request by gavofyork**
Incremental improvement to AZ designed to facilitate more incremental modularity and demonstrate efficacy through implementation of a small but significant part of AZ's functionality.

### CODE REVIEW

File: alethzero/MainFace.h [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+class MainFace: public QMainWindow, public Context
+{
+public:
+	explicit MainFace(QWidget* _parent = nullptr): QMainWindow(_parent) {}
+
+	void adoptPlugin(Plugin* _p);
+	void killPlugins();
+
+	void allChange();
```
Comment (arkpar) [inline alethzero/MainFace.h]: make this an event? onAllChange won't be required

File: alethzero/AllAccounts.h [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ * @date 2015
+ */
+
+#pragma once
+
+#include <QListWidget>
+#include <QPlainTextEdit>
+#include "MainFace.h"
+
+namespace Ui {
```
Comment (arkpar) [inline alethzero/AllAccounts.h]: brace style

File: alethzero/MainFace.h (hunk 1 above)
Comment (gavofyork) [inline alethzero/MainFace.h]: you mean Qt signal?

Comment (arkpar) [inline alethzero/MainFace.h]: yes

**Pull request merged**